stories/Img.stories.tsx

```diff
@@ -170,6 +170,105 @@ export const ProvideZoomImg = (props: typeof Zoom) => (
 
 // =============================================================================
 
+const CustomZoomContentWithLoader: UncontrolledProps['ZoomContent'] = ({
+  img,
+  isZoomImgLoaded,
+  modalState,
+}) => {
+  const [
+    showLoader,
+    setShowLoader,
+  ] = React.useState(!isZoomImgLoaded)
+
+  /**
+   * Delay the loader so the loading spinner is noticeable
+   */
+  React.useEffect(() => {
+    if (modalState === 'LOADING') {
+      setShowLoader(true)
+      if (isZoomImgLoaded) {
+        setTimeout(() => setShowLoader(false), 1000)
+      }
+    }
+  }, [isZoomImgLoaded, modalState])
+
+  return (
+    <>
+      {img}
+      {showLoader && (
+        <div className='zoom-img-loader-wrapper'>
+          <div className='zoom-img-loader' />
+        </div>
+      )}
+    </>
+  )
+}
+
+export const ZoomImgLoader = (props: typeof Zoom) => (
+  <main aria-label="Story">
+    <h1>
+      ZoomImg with Loading State
+    </h1>
+    <div className="mw-600">
+      <p>
+        This example shows how to provide loading feedback when using a high-resolution&nbsp;
+        <code>zoomImg</code>. The <code>ZoomContent</code> component uses the&nbsp;
```

Comment:
Alternatively, I think this works without having to use a non-breaking space character.

```suggestion
        This example shows how to provide loading feedback when using a high-resolution{' '}
        <code>zoomImg</code>. The <code>ZoomContent</code> component uses the{' '}
```